zk-sdk/src/zk_elgamal_proof_program/proof_data/batched_grouped_ciphertext_validity/handles_2.rs

```diff
@@ -95,6 +102,53 @@ impl BatchedGroupedCiphertext2HandlesValidityProofData {
     }
 }
 
+#[cfg(target_arch = "wasm32")]
+#[cfg_attr(target_arch = "wasm32", wasm_bindgen)]
+impl BatchedGroupedCiphertext2HandlesValidityProofData {
```

Comment:
nit: can you just put a comment saying to keep this in sync with the non wasm32 constructor and making it clear that this exists because wasm-bindgen doesn't support generics?